packages/Webkul/Shop/src/Http/Controllers/OrderController.php

```diff
@@ -50,7 +50,13 @@ public function __construct(
     */
     public function index()
     {
-        return view($this->_config['view']);
+        $order = $this->orderRepository->findOneWhere(['customer_id' => auth()->guard('customer')->user()->id]);
```

Comment:
This will always return first customer order. I don't see any purpose of this.

Reply:
You are right, i only tested it with one order, i will provide a fix for this